src/main/kotlin/service/store/StoreService.kt

```diff
@@ -8,12 +8,14 @@ import org.jetbrains.exposed.sql.transactions.transaction
 import org.joda.time.DateTime
 import service.store.data.ThinPokemon
 import service.user.balance.BalanceIncreaseRateManager
+import service.user.balance.BalanceIncreaseRateManager.Companion.IncreaseRateScalingFactor
 import service.user.balance.BalanceManager
 import utility.PokeApi
+import kotlin.random.Random
 
 const val BoosterpackSize = 5
-const val BasePriceFactor = 1.0 / 55.0
-const val LocationIdBaseIncrease = 1.25
+const val LocationIdBaseIncrease = 3.0
```

Comment:
I was wondering why the constants are upper camel case? Java convention is to have them full caps. Is this Kotlin convention?

Reply:
No, not really. I can change it though. I was using this in the past.

Reply:
Ah okay. Don't put too much work in it. Just looks weird.

Reply:
I'm gonna fix it tomorrow or next week since I wanna fix the bugs first.